script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -383,22 +384,22 @@ I am convinced that reducing the severity of the punishment by confessing to one
 [sha:909fd2393bcfdc0faf353faa535071a9b905e468]{
 -- Page 127, Offset 22461.
 -- 「はい。なんでしょうか、遠野くん」
-"Yes? What do you want to say, Tohno-kun?"
+"Yes? What is it, Tohno-kun?"
 }
 [sha:2c9b1b5d0543fe3410218d6b53f7b487e0cf89fd]{
 -- Page 127, Offset 22462.
 -- 「昨夜の話、なんですけど」
-"It's about yesterday night..."
+"About yesterday night..."
 }
 [sha:4102ebad29fa787fa6e9beb5195f1d93073ea233]{
 -- Page 127, Offset 22463.
 -- 「ほう。昨夜のお話ですか。それは興味深い。静聴しますので遠慮なくどうぞ。懺悔だと思って」
-"Oh, about yesterday night, you say. That's very interesting. I'll listen without interrupting you, so say what you want without reserve. Consider it your penance."
+"Oh, about yesterday night, you say. How interesting. I'll listen along quietly, so feel free to speak your mind. Consider it your penance."
 }
 [sha:5cf540d122226f74afd4db1a8fd53c9d1e15b17d]{
 -- Page 128, Offset 22464.
 -- 「……すみません。俺は先輩との約束を破って、アルクェイドと吸血鬼を探しに行きました。アイツとは二日前からそういう約束だったんです」
-"...I'm sorry. I broke my promise to you, Senpai, and I accompanied Arcueid in her search for vampires. That's what I promised to her two days ago."
+"...I'm sorry. I broke my promise to you, Senpai, and I accompanied Arcueid in her search for vampires. I promised her that two days ago."
```

Comment:
I'm sorry. I broke my promise to you, and accompanied Arcueid in her search for vampires.